### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by onyevyerov**

### CODE REVIEW

**Comment by onyevyerov** [comment]
image_2024-09-09_15-48-49 (2)
image_2024-09-09_15-48-49 (3)
image_2024-09-09_15-48-49 (4)
image_2024-09-09_15-48-49 (5)
image_2024-09-09_15-48-49

**Comment by onyevyerov** [comment]
image

File: taxi/urls.py [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+    path("cars/<int:pk>/",
+         CarDetailView.as_view(),
+         name="car-detail"),
+    path("drivers/",
+         DriverListView.as_view(),
+         name="driver-list"),
+    path("drivers/<int:pk>/",
+         DriverDetailView.as_view(),
+         name="driver-detail"),
+
```
Comment (LLkaia) [inline taxi/urls.py:25]: Remove redundant blank line

**Review by LLkaia — COMMENTED** [review]
Tests failed + I see PR closed now

**Pull request closed without merging**